module/VuFind/tests/unit-tests/src/VuFindTest/RecordDriver/EDSTest.php

```diff
@@ -564,17 +564,19 @@ public function testGetLinkedFullTextLink(): void
     }
 
     /**
-     * Test getItemsSubjects for a record.
+     * Test getAllSubjectHeadingsFlattened for a record.
      *
      * @return void
      */
-    public function testGetItemsSubjects(): void
+    public function testGetAllSubjectHeadingsFlattened(): void
     {
         $driver = $this->getDriver('valid-eds-record');
         $this->assertEquals(
-            '<a href="../EDS/Search?lookfor=%22PSYCHOTHERAPY%22&amp;type=SU">PSYCHOTHERAPY</a>, ' .
-            '<a href="../EDS/Search?lookfor=%22METAPHOR%2E%22&amp;type=SU">METAPHOR.</a>',
-            $driver->getItemsSubjects()
+            [
+                '<a href="../EDS/Search?lookfor=%22PSYCHOTHERAPY%22&amp;type=SU">PSYCHOTHERAPY</a>',
```

Comment:
Sure! The only reason I could think of keeping the historical function is in case somebody is relying on it for a local customization, but I suspect that's not especially likely. :-)

Reply:
OK. I am removing the potential dot at the end while I'm at it.